src/main/java/io/curity/identityserver/plugin/alarmhandler/EventsBridgeAlarmConfiguration.java

```diff
@@ -68,6 +67,6 @@ interface AWSAccessMethod extends OneOf
         }
 
         @Description("EC2 instance that the Curity Identity Server is running on has been assigned an IAM Role with permissions to Events Bridge.")
-        Optional<@DefaultBoolean(false) Boolean> isEC2InstanceProfile();
+        Optional<Boolean> isEC2InstanceProfile();
```

Comment:
I updated this to use the more standard syntax from the [DynamoDBDataAccessProvider](https://github.com/curityio/dynamodb-data-access-provider/blob/master/src/main/kotlin/io/curity/identityserver/plugin/dynamodb/configuration/DynamoDBDataAccessProviderConfiguration.kt), to avoid a load error.